### PR TITLE
Yaya/custom recycler view+bottom nav bar+favorites page+expanded list+refresh button+Search functions

### PR DESCRIPTION
**Pull request by lamsalab**
I think this is a minimum viable product. I have tested it on multiple devices and it works fine. I still have more to implement, which will be shipped periodically unlike this one. Let me know what I have to change before we make a merge.

### CODE REVIEW

File: app/src/main/java/edu/grinnell/appdev/events/FragmentMap.java [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+        // Gets the MapView from the XML layout and creates it
+        mapView = (MapView) v.findViewById(R.id.mapview);
+        mapView.onCreate(savedInstanceState);
+
+        if (savedInstanceState != null){
+
+        }
+
+        mapView.getMapAsync(new OnMapReadyCallback() {
```
Comment (mattorib) [inline app/src/main/java/edu/grinnell/appdev/events/FragmentMap.java]: There seems to be an authentication issue which causes the map to not work at all.

File: app/src/main/java/edu/grinnell/appdev/events/AsyncParser.java [hunk 2]
```diff
@@ -19,35 +22,58 @@
...
+    private ProgressDialog progressDialog;
+    private Activity activity;
+
+    private String title;
+    private String content;
+    private String evlocation;
+    private Long startTime;
+    private Long endTime;
+    private String orgEmail;
+    private String organizer;
```
Comment (mattorib) [inline app/src/main/java/edu/grinnell/appdev/events/AsyncParser.java:51]: Unused variable

File: app/src/main/java/edu/grinnell/appdev/events/AsyncParser.java [hunk 3]
```diff
@@ -59,30 +85,42 @@ protected Integer doInBackground(String... strings) {
             e.printStackTrace();
             return ERROR_PARSING;
         }
-
+        int count = 0;
```
Comment (mattorib) [inline app/src/main/java/edu/grinnell/appdev/events/AsyncParser.java]: Unused variable

File: app/src/main/java/edu/grinnell/appdev/events/onParseComplete.java [hunk 4]
```diff
@@ -3,6 +3,7 @@
 
 import java.util.List;
 
+
 /**
```
Comment (mattorib) [inline app/src/main/java/edu/grinnell/appdev/events/onParseComplete.java:9]: Classes and interfaces should be named with upper camelcase, so the first letter should be capitalized.

**Review by mattorib — CHANGES_REQUESTED** [review]
Looks good for the most part! The map needs an authentication token, which you should be able to produce, and the files should be organized a little more (ie. adapters, activities, models, etc. in individual folders).

File: app/src/main/java/edu/grinnell/appdev/events/FragmentMap.java (hunk 1 above)
Comment (lamsalab) [reply app/src/main/java/edu/grinnell/appdev/events/FragmentMap.java]: I tested it and it seems to work on multiple devices.

**Review by zoerosemarie — APPROVED** [review]
Looks good!

**Review by mattorib — APPROVED** [review]

**Pull request merged**